app/views/_cascade/formats/modular/_page_wrapper_open.erb

```diff
@@ -0,0 +1,19 @@
+<%
+
+# VTL variables
+baseClass = ''
+sectionClasses = ''
+currentPagePath = @current_page_path || '/'
+
+currentPagePath.split('/').each do |name|
+  if currentPagePath.split('/').first == name
+    baseClass = name
+    sectionClasses = baseClass
+  elsif currentPagePath.split('/').last != name
+    baseClass = "#{baseClass}__#{name}"
+    sectionClasses = "#{sectionClasses} #{baseClass}"
+  end
+end
```

Comment:
Not sure what's going on in the big picture here, but I'm just curious about what happens if `currentPagePath.split('/').last == name`

I just don't often see a conditional block with a `==` followed by an elseif with `!=`